2018-August-28/MNoterman_latex.tex

```diff
@@ -107,8 +115,8 @@
 neuron death, providing evidence for Ca\textsubscript{v}1.2 as a key regulator
  of neuronal survival in postnatal hippocampal neurogenesis (Figure 1) {[}4{]}.
 
-Cell death is a prominent feature of many forms of neuropsychiatric
-illness {[}31{]} and can result from an array of mechanisms, including
+Cell death is a prominent feature in many forms of neuropsychiatric
+diseases {[}31{]} and can result from an array of mechanisms, including
 necrosis, autophagy, and apoptosis {[}32{]}. 
 Importantly, loss of mitochondrial calcium homeostasis predisposes neurons to activation of
 cell death programs, placing mitochondria as pivotal regulators in
```

Comment:
below sentence (what tissues are not energy demanding?) change to excitable tissue

Reply:
energy-demanding is a common fluff phrase.